doc/source/whatsnew/v0.20.0.txt

```diff
@@ -105,6 +105,7 @@ Other enhancements
   of sorting or an incorrect key. See :ref:`here <advanced.unsorted>`
 
 - ``pd.cut`` and ``pd.qcut`` now support datetime64 and timedelta64 dtypes (:issue:`14714`, :issue:`14798`)
+- ``pd.qcut`` can optionally remove duplicate edges instead of throwing an error (:issue:`7751`)
```

Comment:
``pd.cut/qcut`` have gained the ``duplicates`` kw to control whether to raise on duplicated edges.